### DISCUSSION TITLE
Feature requests, ideas, use cases and wish lists

### DISCUSSION THREAD

**Discussion by sapiderman** [hyperjumptech/monika#1052] [Ideas]
Starting a new discussion on possible features and ideas.

**Comment by sapiderman** [comment]
Moving this idea  from @luizrojo here, because I think it warrants some thought and more discussion. And not on the back of a closed ticket :)

As quoted:

"The idea was to be able to add an actual payload, like XML, YAML, or JSON, directly on the config file.

The ability to reference the payload from a file would be even nicer, since one could use a config management system to template payloads and deliver them through automation."


My questions then, 
1. Where would this payload file be located? A separate path? A full remote url?
2. Is it a static file? Is it autogenerated or generated by some event? A logs perhaps?

**Comment by mike-alesso** [comment]
Being able to reference environment variables would be very helpful, especially for values such as usernames, passwords, api keys and etc.

**Reply by sapiderman** [reply]
upvote on this! this is a great idea!
